fcitx5-fbterm.cpp

```diff
@@ -0,0 +1,365 @@
+/*
+* SPDX-FileCopyrightText: 2021 duzhaokun123 <duzhaokun2@outlook.com>
+*
+* SPDX-License-Identifier: GPL-3.0-or-later
+*
+*/
+
+#include "fcitx5-fbterm.h"
+#include "imapi.h"
+#include "keycode.h"
+#include "keymap.h"
+#include "utils.h"
+#include <cstring>
+#include <fcitx-gclient/fcitxgclient.h>
+#include <fcitx-utils/capabilityflags.h>
+#include <fcitx-utils/keysymgen.h>
+#include <fcitx-utils/fs.h>
+#include <gio/gio.h>
+#include <iconv.h>
+
+#define eprintf(args...) fprintf(stderr, args)
+#define getArg1(i) getArg3(argc, argv, i)
+#define clearWin(id) set_im_window(id, rect0)
```

Comment:
Don't use macro for this. Define a function. function is type safe and error prone, macro is not.